apple/testing/default_runner/ios_xctestrun_runner.template.xctestrun

```diff
@@ -50,5 +48,35 @@ BAZEL_COMMAND_LINE_ARGS_SECTION
 BAZEL_SKIP_TEST_SECTION
 BAZEL_ONLY_TEST_SECTION
   </dict>
+  <key>__xctestrun_metadata__</key>
+  <dict>
+    <key>CodeCoverageBuildableInfos</key>
+    <array>
+      <dict>
+        <key>Architecture</key>
+        <string>BAZEL_ARCHITECTURE</string>
+        <key>BuildableIdentifier</key>
+        <string>000000000000000000000000:primary</string>
+        <key>IncludeInReport</key>
+        <false/>
+        <key>IsStatic</key>
+        <false/>
+        <key>Name</key>
+        <string>BAZEL_TEST_BUNDLE_NAME</string>
+        <key>ProductPath</key>
+        <string>FakePath</string>
+        <key>SourceFiles</key>
+        <array></array>
+        <key>SourceFilesCommonPathPrefix</key>
+        <string>BAZEL_SOURCE_FILES_COMMON_PREFIX/</string>
```

Comment:
It _seems_ like these two can be removed. Might be nice to exclude `SourceFiles` since it's empty/not correct at this point if it actually works. Let me know how it goes in your testing.

Reply:
That's a good idea. I removed these and tested locally and everything worked the same as before. Nice to not have incorrect information 🙂